### DISCUSSION TITLE
Problems saving the additional fields in the backoffice clients

### DISCUSSION THREAD

**Discussion by 170686** [PrestaShop/PrestaShop#31696] [Q&A] [answered]
Good morning,

we are having problems saving the additional fields in the backoffice clients on the dilorenzostore.com site, as we see the error visible in the following link and in the following screenshot:

https://www.dilorenzostore.com/Backoffice/index.php?controller=AdminGcustomfieldrequest&token=857bf064ec81257ce671068b6ae0ba2c

![image](https://user-images.githubusercontent.com/47886531/223976982-8f2324b9-df3e-46ca-a1ed-365edfdb433a.png)

Could you tell us how to solve the problem?

Thanks

**Comment by kpodemski** [comment]
Hello @170686 

Unfortunately, there's no way somebody can help only by looking at the white page with a JSON response. It seems like a third-party module. You should contact the author of this solution.

**Comment by 170686** [comment]
Good morning Krystian,

Could you tell us what you need in detail to be able to verify in depth
since we urgently need to resolve the problem?

Thank you




  CARLO MONETTAWeb Developer
wa. +39 340 738 9116
*lenusmedia.com <http://lenusmedia.com>*


Il giorno gio 9 mar 2023 alle ore 22:15 Krystian Podemski <
***@***.***> ha scritto:

**Comment by 170686** [comment]
Good evening Krystian,

Is there any news regarding our requests since we urgently need to resolve
the problem?

Thank you




  CARLO MONETTAWeb Developer
wa. +39 340 738 9116
*lenusmedia.com <http://lenusmedia.com>*


Il giorno ven 10 mar 2023 alle ore 10:26 Carlo Monetta <
***@***.***> ha scritto:

**Comment by kpodemski** [comment, marked as answer]
Hello @170686 

Remember that it is not a support center. You should ask the author of the module for help.

To me, it looks like you have some javascript errors, and something that should be dynamically saved (via ajax) is redirecting you to the new page.

**Comment by 170686** [comment]
Hi Krystian,

Could you tell us in detail what went wrong in Ajax that led to generating
the reported problem?

Thank you




  CARLO MONETTAWeb Developer
wa. +39 340 738 9116
*lenusmedia.com <http://lenusmedia.com>*


Il giorno lun 13 mar 2023 alle ore 18:29 Krystian Podemski <
***@***.***> ha scritto:

**Reply by kpodemski** [reply]
Please DO NOT share your credentials on the public forum. You should change them as soon as possible.

**Comment by kpodemski** [comment]
Hi, I can't tell you that. It'd require me to look into your website sources, which is impossible.

**Comment by 170686** [comment]
Good morning Krystian,

Could you update us on our request on where to send you the FTP access
credentials of the dilorenzostore.com site?

Thank you




  CARLO MONETTAWeb Developer
wa. +39 340 738 9116
*lenusmedia.com <http://lenusmedia.com>*


Il giorno mar 14 mar 2023 alle ore 09:30 Carlo Monetta <
***@***.***> ha scritto:
